common/utils/filters.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import { nodeListValueToArray, inputValue } from '@weco/common/utils/forms';
+import { LinkProps } from '@weco/common/model/link-props';
+import { RadioGroupOption } from '@weco/common/views/components/RadioGroup/RadioGroup';
+import {
+  worksLink,
+  imagesLink,
+  WorksRouteProps,
+} from '@weco/common/services/catalogue/ts_routes';
+
+export const getNonEmptyFilters = (
```

Comment:
I think we're still conflating name here. We have aggregations, which is the data we get from the API, and we turn them into filters, which is the UI.


```suggestion
export const getNonEmptyAggregations = (
```

?